### DISCUSSION TITLE
Could you make cursor color auto change?

### DISCUSSION THREAD

**Discussion by CatCatCatDog** [microsoft/terminal#13440] [Ideas]
![1](https://user-images.githubusercontent.com/23658192/177537316-86adc9d4-5540-40db-a74d-aba9e32bbcab.png)
Sometimes confuse me, If my cursor access to green word.
Could you make cursor color automatically change? If my cursor access to similar color?

**Comment by zadjii-msft** [comment]
What would you like it to change to in this scenario? We've got a pile of open feature requests for modifying the cursor's appearance based on the text color, so I want to make sure what you're looking for is covered by those requests.

**Reply by CatCatCatDog** [reply]
I remember when I use iterm2, switch color happens, If cursor access to similar color.
i'm not sure 100%. but, Please, make like iterm2. iterm2 is very convenience.
